Automattic/wp-calypso
# Pricing page: implement most popular experiment

### Changes proposed in this Pull Request

This PR implements the A/B experiment that replaces the plan cards row by the most popular products, in the pricing page. It also makes the ascending prices experiment the new default.

Fixes 1196341175636977-as-1199508074334637

### Implementation notes

Implementation isn't clean. Fortunately the lifespan of the experiment is quite short.

### Testing instructions

_Prerequisite: you'll need to change the variation of the `popularProductsOffering` experiment shown in the capture below:_
<img width="546" alt="Screen Shot 2020-12-15 at 10 15 44 AM" src="https://user-images.githubusercontent.com/1620183/102235683-95097500-3ec0-11eb-8c8d-5c1a33e5924f.png">

- Download the PR
- Run Calypso or Jetpack cloud
- Visit the pricing page
- Check that you see the new default page. Order of the plan cards row should now be: Security Daily, Security Real-time, Complete
- Switch to the `dailyProducts` variation
- Check that the order of cards, the 2 section headings, and features (in the top row only) match the copy and mockups (see linked task, and captures below). The case of the features strings can be different to be consistent with the rest of the page.

### Screenshots

_control_
<img width="1077" alt="Screen Shot 2020-12-15 at 10 16 04 AM" src="https://user-images.githubusercontent.com/1620183/102236157-27117d80-3ec1-11eb-91a1-0d85cb0ad323.png">
<img width="1100" alt="Screen Shot 2020-12-15 at 10 16 11 AM" src="https://user-images.githubusercontent.com/1620183/102236167-2a0c6e00-3ec1-11eb-925a-c84314281484.png">



_dailyProducts_
<img width="1075" alt="Screen Shot 2020-12-15 at 10 15 13 AM" src="https://user-images.githubusercontent.com/1620183/102236133-1d881580-3ec1-11eb-9747-833c5ce7db23.png">

<img width="1133" alt="Screen Shot 2020-12-15 at 10 15 23 AM" src="https://user-images.githubusercontent.com/1620183/102236142-22e56000-3ec1-11eb-8073-23143f8a171f.png">
